iceoryx_examples/icedelivery/iox_publisher_untyped.cpp

```diff
@@ -58,17 +58,16 @@ int main()
             RadarObject* data = new (chunk) RadarObject(ct, ct, ct);
             iox::cxx::Expects(chunk == data);
 
-            data->x = 1.0;
-            data->y = 2.0;
-            data->z = 3.0;
+            data->x = ct;
+            data->y = ct;
```

Comment:
could we differ the values somehow, maybe like this:
```cpp
data->x = ct;
data->y = ct * ct;
data->z = ct + 2.5;
```

so that you can distinct them in the output.

Reply:
I'd keep things simple here and not distract the users. `RadarObject` is used to show a complex data type can be transmitted, instead of just transmitting `int`'s.